### DISCUSSION TITLE
Multiple workspaces

### DISCUSSION THREAD

**Discussion by winkelement** [outline/outline#4531] [Self Hosting] [answered]
I wasn't able to find any information about [this feature](https://www.getoutline.com/changelog/multiple-workspaces) in a self hosted environment and would like to enable that very much. Anyone can give me a hint?

Thank you very much!

System Info: Using `outlinewiki/outline` Docker Image (0.66.2)

**Comment by tommoor** [comment, marked as answer]
Hi Robert, this feature does not exist in the community edition – we plan on bringing it to the Enterprise edition in the future but for now it's only part of the cloud-hosted platform.

**Reply by winkelement** [reply]
That's a pity but for sure i understand you need to keep features exclusive for paying customers.
Thank you very much for the quick answer!
